tests/userProfileTest.js

```diff
@@ -0,0 +1,173 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+import Chai from 'chai';
```

Comment:
change  'Chai' to lowercase

Reply:
Noted, i'll change it as soon as possible